### PR TITLE
Optimize database configuration for mysql5.6.

### PR DESCRIPTION
**Pull request by engraving-knife**
## Why are the changes needed?
Avoid service startup failure due to indentation errors when modifying database configuration.
Modify the mysql sql file so that it can run in mysql5.6.

## Brief change log
- Modify the database default configuration indent size in config.yaml.
- Modify the table creation statement and change the row format of some table to DYNAMIC

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (no)
- If yes, how is the feature documented? (not applicable)

### CODE REVIEW

**Comment by zhoujinsong** [comment]
@engraving-knife Thanks for reporting and trying to fix the issue.

Can you help to create an issue and link to this PR to help others find the related issue.

File: amoro-ams/amoro-ams-server/src/main/resources/mysql/ams-mysql-init.sql [hunk 1]
```diff
@@ -82,7 +82,7 @@ CREATE TABLE `table_identifier`
     `format`          VARCHAR(32)  NOT NULL COMMENT 'Table Format',
     PRIMARY KEY (`table_id`),
     UNIQUE KEY `table_name_index` (`catalog_name`,`db_name`,`table_name`)
-);
+) ROW_FORMAT=DYNAMIC;
```
Comment (zhoujinsong) [inline amoro-ams/amoro-ams-server/src/main/resources/mysql/ams-mysql-init.sql:85]: Maybe add engine and charset information for this table too:
```suggestion
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COMMENT 'Table identifier for AMS' ROW_FORMAT=DYNAMIC;
```

**Review by zhoujinsong — COMMENTED** [review]
I left a small suggestion, PTAL.

**Review by czy006 — APPROVED** [review]
LGTM

**Pull request merged**